src/ValueFormatters/QuantityFormatter.php

```diff
@@ -38,6 +38,12 @@ class QuantityFormatter extends ValueFormatterBase {
 	const OPT_APPLY_ROUNDING = 'applyRounding';
 
 	/**
+	 * Option key controlling whether the quantity's unit of measurement should be included
+	 * in the output.
+	 */
+	const OPT_APPLY_UNIT = 'applyUnit';
```

Comment:
Not when generating a value for use in an input field. We cannot parse localized units (yet). So in case the unit is to be input separately, it must not be present in the text representation of the quantity.